### ISSUE TITLE
Creating web users is broken

### ISSUE THREAD

**Issue by bsharif** [WDDCP/commcare-wddcp#12]
Creating web users is currently broken.
This is done through: Users > Web Users & Roles 

Note that 'Web Users' are different to 'Mobile Workers'
Mobile workers: Login to the app and collect data
Web Users: Login to the app design pages and manage users/apps etc
(Due to these differences I'm not sure if this issue would be linked to #10 or something seperate)

The current error is seen on clicking the Web Users & Roles link. 
Takes me to this [page](http://ec2-35-177-33-162.eu-west-2.compute.amazonaws.com:8000/a/demo/settings/users/web/):
![fireshot capture 10 - programmingerror at _a_demo_settings_u_ - http___ec2-35-177-33-162 eu-west-2](https://user-images.githubusercontent.com/7132252/27769575-9ee6868c-5f24-11e7-8fd1-368213c3b309.png)

**Comment by bsharif** [comment]
Dev group thread [here](https://groups.google.com/d/msg/commcare-developers/VFCi6wnsnKs/mlDIY309BgAJ)

Can you make out anything useful from that page Helen?

**Comment by lamby** [comment]
Probably need to run ``./manage.py migrate`` or otherwise work out why running this command is not creating this database table (postgresql calls tables "relations").

**Comment by DeckOfPandas** [comment]
Not quite fixed, but easy to sort for local builds. Advice on setting up django environment is now in README.md.

tl;dr: Needed to `./manage.py makemigrations` then `./manage.py migrate` (inside the docker imgae) first.

Closing this, and will open a new issue about making sure this is taken care of in the deployment.
